gidallocator/gidallocator.go

```diff
@@ -210,6 +218,17 @@ func (a *Allocator) collectGids(className string, gidTable *allocator.MinMaxAllo
 		}
 	}
 
+	// if a gid reclaimer was given, call it to allow it to contribute additional GIDs to the gidtable
+	if a.gidReclaimer != nil {
+		glog.Info("Calling GID reclaimer")
```

Comment:
The casing of the logs does not match the casing of other logs in this file.

Further, there are very few logs at all and no `Info` or even `Debug` logs in this file. I tend to follow the logging style of the project I'm changing as closely as possible. I would probably drop the `Info` logs but that's up to you.

Reply:
Done